src/test/java/org/traccar/protocol/MegastekProtocolDecoderTest.java

```diff
@@ -95,6 +101,9 @@ public void testDecode() throws Exception {
         verifyPosition(decoder, text(
                 "LOGSTX,123456789012345,$GPRMC,230739.000,A,3841.81895,N,09494.12409,W,0.00,0.00,270914,,,A*70,L,,imei:123456789012345,0/7,269.7,Battery=100%,,0,,,5856,78A3;78"));
         
+        Position testPosition = (Position) decoder.decode(null, null, "$MGV002,860719020193193,DeviceName,R,240214,104742,A,2238.20471,N,11401.97967,E,00,03,00,1.20,0.462,356.23,137.9,1.5,460,07,262C,0F54,25,0000,0000,0,0,0,28.5,28.3,,10,100,Timer;!");
```

Comment:
We have a special method `verifyAttribute`. See other tests for examples.

Reply:
Ah right, I knew there had to be a better way! Fixed that. :)